core/src/main/java/io/onetable/delta/DeltaSchemaExtractor.java

```diff
@@ -92,6 +92,7 @@ private DataType convertFieldType(OneField field) {
         return DataTypes.LongType;
       case BYTES:
       case FIXED:
+      case UUID:
```

Comment:
I think the metadata will be empty for no other logic here, not sure whether this is correct